### ISSUE TITLE
Installing driver

### ISSUE THREAD

**Issue by ihatetn931** [nefarius/ScpToolkit#238]
Driver installer won't install, windows 8.1, Goes through setup process then when it ask me to run the driver installer i get this.
`A fatal error occured. Consult the logs for details`

http://pastebin.com/sRc3cuZ2

**Comment by ihatetn931** [comment]
even if i copy the missing files to where it's trying to load from it still gives me the same issue. There isn't even an x86 folder so i dunno why it's trying to load files from a folder that don't exist

**Comment by ihatetn931** [comment]
I got this issue figured out for some reason my os was running apps in 32 bit instead of 64, but i've ran in to another issue, it picks up the controller just fine but nothing works on it like buttons ect. it's like the virtual joystick is not reading the input of the actuall controller, when i try my sixaxis that isn't a dualshock 3 it works but when i use the controller i've been using for ages on the pc it dosen't work, i choose to use this one cause the rumble works in windows. I've tried diffrent usb wires, also checked to see if the controller still works and it works great on the ps3. My controller just stoped workin the other day and i have no idea why nothing has changed on my pc, played one game prolly over the weekend then went to go play another game and all controller input wasn't working (this was on scarlet crush drivers) so i decided to try these ones to see if that was the issue.
